TalkAloud/TalkAloud/Managers/AudioManager.swift

```diff
@@ -31,7 +31,11 @@ class AudioManager {
         
         didNewRecording = true
         
-        audioRecording = CoreDataManager.sharedInstance.createNewAudioRecording(uniqueFileName: uniqueFileName)
+        if isDirectoryCreated() {
+            audioRecording = CoreDataManager.sharedInstance.createNewAudioRecording(uniqueFileName: uniqueFileName)
+        } else {
+            audioRecording = CoreDataManager.sharedInstance.createNewAudioRecording(uniqueFileName: uniqueFileName)
```

Comment:
What's the difference between the if and else statements?
Because they have the same logic lol

Reply:
Well if I return false in my createDirectory then I would go to the else case.

Reply:
I still want my audioRecording initialized whether the directory needs to be created or not.

Reply:
Alright, I changed it to where I just call the method since it determines whether the directory has been created or not.